src/main/java/com/textmessenger/service/DialogServiceImpl.java

```diff
@@ -22,32 +18,27 @@
   private final DialogRepository dialogRepository;
   private final UserRepository userRepository;
   private final NotificationService notificationService;
-  private final DialogMapper dialogMapper;
-  private final UserMapper userMapper;
 
   public DialogServiceImpl(DialogRepository dialogRepository, UserRepository userRepository,
-                           NotificationService notificationService, DialogMapper dialogMapper,
-                           UserMapper userMapper) {
+                           NotificationService notificationService) {
     this.dialogRepository = dialogRepository;
     this.userRepository = userRepository;
     this.notificationService = notificationService;
-    this.dialogMapper = dialogMapper;
-    this.userMapper = userMapper;
   }
 
   @Override
-  public void createDialog(DialogRxDto dialog) {
-    dialogRepository.save(dialogMapper.dialRxDtoToDial(dialog));
+  public void createDialog(Dialog dialog) {
+    dialogRepository.save(dialog);
   }
 
-  public List<DialogTxDto> getDialogsByUser(UserRxDto user) {
-    return dialogMapper.dialsToDialTxDtos(
-            dialogRepository.findDialogsByUsers(userMapper.userRxDtoToUser(user)));
+  public List<DialogToFront> getDialogsByUser(User user) {
+    return DialogToFront.convertDialogsListToResponse(dialogRepository.findDialogsByUsers(user));
+    // return dialogRepository.findDialogsByUsers(user);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=158&issues=AWWU5_bPAJjEPbGfqJZQ&open=AWWU5_bPAJjEPbGfqJZQ)

[ISSUE_KEY]: AWWU5_bPAJjEPbGfqJZQ